lib/performance_test.rb

```diff
@@ -56,6 +56,21 @@ def get_script_user
     user
   end
 
+  def deploy_app(app, deploy_params = {})
+    # Override distribution bits to ensure no whole-corpus streaming
+    # searches have to visit 64k buckets, but only 2.
```

Comment:
Consider removing this comment, since it only makes sense with 1 distribution bit